tox.ini

```diff
@@ -6,6 +6,7 @@ deps =
     pytest
     pytest-aiohttp
     pytest-cov
+extras = web-poet
```

Comment:
We could have separate tests that test thje main functionality without the PO deps. Not sure if we should though. I guess it could catch some stray imports in the main code paths.

Reply:
Yes, it would be nice to have at least 1 job run without the extra.